packages/analytics/analytics-chart/src/components/TopNTable.vue

```diff
@@ -149,14 +149,14 @@ const displayRecord = computed(() => {
     return {}
   }
 
-  return props.data.meta.display[displayKey.value]
+  return props.data.meta.display[displayKey.value as QueryableExploreDimensions]
```

Comment:
Arrgh.  The cast here is necessary because `displayKey` is a string rather than a `QueryableExploreDimensions | ExploreAggregations`.  `displayKey` can be `''` in addition to a dimension or aggregation, so the cast is wrong, but TS doesn't seem to have any good way to represent "this value might not exist in the object", so perhaps we leave it alone for now.

Reply:
Meaning just go back to have the key in DisplayBlob just be [key: string] ?

Reply:
Ah, I'm sorry.  I actually wasn't asking for a change here, just trying to figure out whether there would be a better way in the future.  It's frustrating when TS can't quite figure out what line to walk between actual type safety and convenience.